pages/[...slug].vue

```diff
@@ -1,18 +1,7 @@
 <template>
   <main>
-    <Layout
-      :title="title"
-      :table-of-contents="tableOfContents"
```

Comment:
I've localised these props inside the Layout component because we weren't using them anywhere else